apps/explorer/lib/explorer/chain/cache/gas_price_oracle.ex

```diff
@@ -358,4 +380,11 @@ defmodule Explorer.Chain.Cache.GasPriceOracle do
   end
 
   defp async_task_on_deletion(_data), do: nil
+
+  defp get_average_block_time do
+    case AverageBlockTime.average_block_time() do
+      {:error, _} -> nil
```

Comment:
```suggestion
      {:error, :disabled} -> nil
```

Let's indicate exact value for the 2nd item of the tuple? Seems, no other `{:error, _}` cases can be.